firebase-firestore/src/main/java/com/google/firebase/firestore/local/QueryEngine.java

```diff
@@ -61,29 +61,35 @@
  */
 public class QueryEngine {
   private static final String LOG_TAG = "QueryEngine";
-  private static final int MIN_COLLECTION_SIZE_TO_AUTO_CREATE_INDEX = 100;
+
+  private static final int INDEX_AUTO_CREATION_DEFAULT_MIN_COLLECTION_SIZE = 100;
```

Comment:
nit: Move "DEFAULT" to the front of `INDEX_AUTO_CREATION_DEFAULT_MIN_COLLECTION_SIZE`, to be consistent with the naming of `DEFAULT_RELATIVE_INDEX_READ_COST_PER_DOCUMENT` (to improve readability).